kotlinx-coroutines-debug/README.md

```diff
@@ -22,6 +22,43 @@ dependencies {
 }
 ```
 
+### Using in unit tests
+
+For JUnit4 debug module provides special test rule, [CoroutinesTimeout], for installing debug probes
+and dump coroutines on timeout to simplify tests debugging.
+
+Its usage is better to demonstrate by the example (runnable code is [here](test/TestRuleExample.kt)):
+ 
+```kotlin
+class TestRuleExample {
+    @Rule
+    @JvmField
+    public val timeout = CoroutinesTimeout.seconds(1)
+
+    private suspend fun someFunctionDeepInTheStack() {
+        withContext(Dispatchers.IO) {
+            delay(Long.MAX_VALUE)
+            println("This line is never executed")
+        }
+
```

Comment:
Consider avoiding extra empty lines in documentation examples and otherwise make them more compact. It makes documentation much easier to grok. Here, I would also consider removing the above `println` and replacing it with end-of-line comment.